### DISCUSSION TITLE
Change SMTP setting (Grafana)

### DISCUSSION THREAD

**Discussion by ascheufl** [NetApp/harvest#3193] [General]
How can I change the SMPT settings for Grafana? I can't find any grafana.ini

**Comment by rahulguptajss** [comment]
@ascheufl Which type of Harvest installation are you using? rpm,deb,native,nabox,container?

**Reply by ascheufl** [reply]
Hi,

I use nabox

**Reply by rahulguptajss** [reply]
Thanks. Is it NABox3 or NABox4?

@ybizeul can help

**Reply by ascheufl** [reply]
NAbox 3.3

**Reply by ybizeul** [reply]
That's in NAbox web UI, under Settings you have a "Email" configuration.

**Reply by ascheufl** [reply]
There are no email settings in the Nabox dashboard

**Reply by ascheufl** [reply]
In Settings is Preferences,Network,Password,LDAP,SSL and Maintenace

**Reply by ybizeul** [reply]
Ah yes, that's a more recent addition, you should upgrade to 3.5.4.

Better yet, migrate to 4.0.7 !

**Reply by ascheufl** [reply]
Thanks for your answer.

**Reply by ascheufl** [reply]
We have updated to 4.0.8 and Ontap provides all data on Grafana. 
But now I have a problem to connecting the StorageGrid. We have created a User (read only) in Storage Grid. 
When I try to connect the Storage Grid, an error comes up: Error while adding system
json: cannot unmarshal object into Go struct field .data of type string 

Do you have any idea ??

Andi

**Reply by ybizeul** [reply]
Please upload a support bundle https://upload.nabox.org/yypu-kaso-qymu

**Comment by ascheufl** [comment]
Done and uploaded.

Am Mo., 14. Okt. 2024 um 10:07 Uhr schrieb Yann Bizeul <
***@***.***>:

**Reply by ybizeul** [reply]
Can you run `curl -k https://<grid_ip>/api/versions` ?

**Reply by ascheufl** [reply]
apiVersion":"3.5","data":[2,3]}

**Reply by ybizeul** [reply]
`curl -k -X POST -d '{"username":"<username>","password":"<password>"}'  https://<grid_ip>/api/v3/authorize` (Please do not paste "data" from result here on github but obfuscate it)

**Reply by ascheufl** [reply]
{"message":{"text":"Invalid request.","key":"error.400","developerMessage":"Parsing JSON body failed."},"code":400,"status":"error","responseTime":"2024-10-14T08:49:50.600Z","apiVersion":"3.5","data":{}}

**Reply by ybizeul** [reply]
I believe you did a mistake in the query

**Reply by ascheufl** [reply]
i can't find any error.... I also tried our Admin User and Password....hmmm

**Reply by ascheufl** [reply]
C:\> curl -k -X POST -d '{"username":"********","password":"*********"}'  https://172.**.**.**/api/v3/authorize

**Reply by ybizeul** [reply]
Interesting. What is that IP address you're using, is that the API gateway ?

**Reply by ascheufl** [reply]
yes

**Reply by ybizeul** [reply]
Ok please send an email to help@nabox.org we can do a remote session

**Comment by ascheufl** [comment]
i send you a email from my business account

Am Mo., 14. Okt. 2024 um 12:33 Uhr schrieb Yann Bizeul <
***@***.***>:

**Comment by ascheufl** [comment]
it works and I found the error. It was a firewall access problem. sorry but
thank you for your help

Am Mo., 14. Okt. 2024 um 12:55 Uhr schrieb Andreas Scheufler <
***@***.***>:

**Reply by cgrinds** [reply]
Glad you got it working @ascheufl. Thanks for the help @ybizeul

**Reply by ybizeul** [reply]
If you have any more informations on the context, I could try and improve NAbox for better diagnostics messages.
